src/editor/classes/AppManager.js

```diff
@@ -0,0 +1,121 @@
+//"use strict";
+/**
+ * @brief This is our controller class.  All calls are made through here
+ *        so we have a consistent interface for javascript communication.
+ * 
+ * Some brief examples of how to use this:
+ * App.proxy.sendEvent(Message, Data);
+ * App.eventHook.onChange();
+ *
```

Comment:
Add more information... for example why and when should I call `App.eventHook.onChange()` or `App.proxy.sendEvent(Message, Data);`.

You can use realistic examples to give the idea.